public/displayUsername.js

```diff
@@ -0,0 +1,14 @@
+function displayUsername() {
+  fetch("/username")
+    .then((response) => {
+      return response.json();
+    })
+    .then((data) => {
+      console.log(data);
+      const username = document.getElementById("account-username");
+      username.textContent += data.username;
+    })
+    .catch((error) => {
+      throw new Error(error);
+    });
```

Comment:
Also, what happens here if the user's not logged in?

Reply:
Currently, the text will not be updated and we throw an error, I think this can be fixed with something like the `checkStatus` function that I used before for the APOD api, I'll have a go implementing this :)